troubleshooting/tools/tracevol.py

```diff
@@ -0,0 +1,300 @@
+"""
```

Comment:
license?

And maybe `#!/usr/bin/python` as the first line (+ make executable)?